SKILL.tsv

```diff
@@ -4843,11 +4843,11 @@ SKILL_20150717_004842	$The instant death chance of [Turn Undead] is proportional
 SKILL_20150717_004843	{memo X}Damage that will be reflected off from [Iron Skin] will be added 20% of Physical ATK per Attribute level.
 SKILL_20150717_004844	$Enemies hit by [Palm Strike] will be affected by [Bleeding] for 8 seconds. Increases the duration of [Bleeding] by 1 second per attribute level. Bleeding damage is proportional to your physical attack.
 SKILL_20150717_004845	$Craft a dispeller to prevent from getting harmful effects. Craft Dispeller is only possible in Rest Mode.
-SKILL_20150717_004846	When you endow the effect that is registered at [Spell Shop] to your colleague, the duration will be increased. Duration will increase by 3 mins per Attribute level.
+SKILL_20150717_004846	When you endow the effect that is registered at [Spell Shop] to your ally, the duration will be increased. Duration will increase by 3 minutes per attribute level.
 SKILL_20150717_004847	$Nearby enemies receive Holy damage equal to the attribute level when using [Arcane Energy]
 SKILL_20150717_004848	$Increases the duration of [Forecast] by 1 second per attribute level.
 SKILL_20150717_004849	$A monster that is transformed by [Change] will have the Item Jackpot Effect with a 0.05% chance per attribute level.
-SKILL_20150717_004850	The hand equipped the shield and dagger of party member
+SKILL_20150717_004850	$Enlarge your party member's offhand to increase attack if it is equipped with a shield or dagger.
```

Comment:
Enlarge your party member's offhand to increase attack if it is equipped with a shield or dagger.

to

Temporarily increases the attack and defense of you and your party member by enlarging the offhand if it is equipped with a shield and dagger.
 
or

Enlarge you and your party member's offhand to temporarily increase attack and defense if it is equipped with a shield and dagger.